tools/slims.py

```diff
@@ -20,20 +20,17 @@ class SlimsSample:
     """
     Class containing the sample data.
```

Comment:
Will this code be exported to a separate repo?

Reply:
Yes, this will end up in [slims_link](https://github.com/ClinicalGenomicsGBG/slims_link) most likely